### ISSUE TITLE
This repository is used by scammers to steal credit card information

### ISSUE THREAD

**Issue by pkarlik** [muhammederdem/vue-interactive-paycard#64]
This repository is used by scammers to steal credit card information and you are part of this scam whether you know or not.
Your repository is hosting the files used directly by the scammer websites. I suggest you to take this repository down. 
I have reported this to the Github as well.

**Comment by sheakhasan926** [comment]
Online banking

**Comment by sheakhasan926** [comment]
Online banking

**Comment by titusdecali** [comment]
@pkarlik Yeah, let's take down every piece of open source code that currently is or could be used by scammers.

Good bye internet, and thanks for all the fish.

**Comment by ghost** [comment]
Yeah actually just found this repository by looking through a scammer's code, atleast they were nice enough to give credit where it's due 😀 anyways this code is currently circulating Denmark and stealing credit card info, with some additional comments in Russian. Very interesting

**Comment by ralucado** [comment]
I also found this repo by looking at a scam website 😂  Still not a reason to take down an open source project

**Comment by matejtu** [comment]
scammers are the problem. we need to take them down. NOT THE REPOSITORY.... the free repository is the hard work of somebody that gives you something for free. something that you maybe dont know how to code, dont have time to code.... SO BE GRATEFUL!!!

Thanks to the developer. great idea and coding. :)
